e2e/content-services/sso/sso-download-directive-component.e2e.ts

```diff
@@ -0,0 +1,160 @@
+
+            silentLogin = false;
+            implicitFlow = true;
+            settingsPage.setProviderEcmSso(TestConfig.adf.url, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, silentLogin, implicitFlow, 'alfresco');
+            loginSsoPage.clickOnSSOButton();
+            loginSsoPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+            navigationBarPage.clickContentServicesButton();
+            contentServicesPage.checkAcsContainer();
+            contentListPage.doubleClickRow(folderName);
+            contentListPage.waitForTableBody();
+            done();
+        });
+
+        afterAll(async (done) => {
+            try {
+                await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+                await uploadActions.deleteFilesOrFolder(this.alfrescoJsApi, folder.entry.id);
+            } catch (error) {
+            }
+            await this.alfrescoJsApi.logout();
+            browser.executeScript('window.sessionStorage.clear();');
+            browser.executeScript('window.localStorage.clear();');
+            done();
+        });
+
+        afterEach(async (done) => {
+            browser.refresh();
+            contentListPage.waitForTableBody();
+            done();
+        });
+
+        it('[C291936] Should be able to download a file', async (done) => {
+            contentListPage.selectRow(pngFileModel.name);
+            contentServicesPage.clickDownloadButton();
+            browser.driver.sleep(1000);
+            expect(Util.fileExists(downloadedPngFile, 30)).toBe(true);
+            done();
+        });
+
+        it('[C291938] Should be able to open a document', async (done) => {
+
+            contentServicesPage.doubleClickRow(firstPdfFileModel.name);
+            viewerPage.checkFileIsLoaded();
+            viewerPage.checkFileNameIsDisplayed(firstPdfFileModel.name);
+            viewerPage.clickCloseButton();
+            contentListPage.waitForTableBody();
+            done();
+        });
+
+        it('[C291942] Should be able to open an image', async (done) => {
+
+            viewerPage.viewFile(pngFileModel.name);
+            viewerPage.checkImgViewerIsDisplayed();
+            viewerPage.checkFileNameIsDisplayed(pngFileModel.name);
+            viewerPage.clickCloseButton();
+            contentListPage.waitForTableBody();
+            done();
+        });
+
+        it('[C291941] Should be able to download multiple files', async (done) => {
+
+            contentServicesPage.clickMultiSelectToggle();
+            contentServicesPage.checkAcsContainer();
+            contentListPage.clickAllRowsCheckbox();
+            dataTableComponent.checkRowIsSelected('Display name', pngFileModel.name);
+            dataTableComponent.checkRowIsSelected('Display name', firstPdfFileModel.name);
+            contentServicesPage.clickDownloadButton();
+            browser.driver.sleep(1000);
```

Comment:
is this sleep really needed?
The method Util.fileExists already waits and repeats until file is downloaded